examples/touchscreen_colorwheel/main/main.c

```diff
@@ -0,0 +1,83 @@
+/*
+ * SPDX-FileCopyrightText: 2022 Espressif Systems (Shanghai) CO LTD
+ *
+ * SPDX-License-Identifier: CC0-1.0
+ */
+
+#include "esp_log.h"
+#include "bsp/esp-box.h"
+#include "lvgl.h"
+
+#define TAG "ESP-BOX"
+#define APP_DISP_DEFAULT_BRIGHTNESS 50
```

Comment:
Why not 100?

FYI: Helper function `bsp_display_backlight_on(void);` can be used to set brightness to 100%